src/Components/ScrivitoExtensions/ColumnsEditorTab.js

```diff
@@ -7,7 +7,7 @@ import { ColumnWidget } from "../../Widgets/ColumnWidget/ColumnWidgetClass";
 export function ColumnsEditorTab({ widget }) {
   const includedWidgetIds = calculateContentIds(calculateContents(widget));
   const uiContext = Scrivito.uiContext();
-  if (uiContext === undefined) return null;
+  if (uiContext === null) return null;
```

Comment:
Seeing this and the documentation of `uiContext` I think we can simplify this even further:

```suggestion
  if (uiContext) return null;
```

Even if `uiContext` would be an empty string the function would now not render instead of a rendering a dysfunctional `scrivito_` css classname.